main.js

```diff
@@ -1 +1,105 @@
-// write your code here to make the tests pass
+function Library () {
```

Comment:
There's nothing wrong with using a function declaration here as opposed to a function expression (i.e. `var Library = function(){}` but for consistency sake I would create both the `Library` and `Book` modules the same way. You create the `Book` function as a function expression.